### DISCUSSION TITLE
Automatic REST API Generation Apollo?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [apollographql/apollo-server#4963] [General]
Is there any way to get an automatically generated REST API with my Apollo GraphQL Server? Also are there maybe any plants to have a feature like this implemented in Apollo?

**Comment by glasser** [comment]
I don't think exposing a REST version of your graph is planned for Apollo Server itself; Apollo Server focuses on being an excellent GraphQL server. It does sound like an interesting thing to add to your web server!
